src/utils/compileManager.ts

```diff
@@ -121,13 +122,16 @@ export class CompileManager {
                     // get doc by fileUri
                     vscode.workspace.openTextDocument(fileUri)
                     .then((doc) => {
-                        if (vscode.window.activeTextEditor) {
-                            let selections = vscode.window.activeTextEditor.selections;
-                            selections = selections.map((sel, index) => {
-                                return index===0?
-                                new vscode.Selection(line,column,line,column)
-                                : sel;
-                            });
+                        for (const editor of vscode.window.visibleTextEditors) {
+                            if (editor.document.uri.toString()===fileUri.toString()) {
+                                editor.selections = editor.selections.map((sel, index) => {
+                                    return index===0?
+                                    new vscode.Selection(line-1,column+1,line-1,column+1)
```

Comment:
It seems like Overleaf always return `-1` for column number, maybe you can try precise column number with the clicked text match